ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1264,6 +1267,7 @@ router_ospf:
     < process_id >:
       passive_interface_default: < true | false >
       router_id: < IPv4_address >
+      log_detailed_adjacency_changes:
```

Comment:
update key to match the cli syntax to avoid confusion.
`log_adjacency_changes_detail: < true | false >`

Reply:
Updates as requested.